.drone/pipelines/default/steps/test/cf_acceptance_tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+# shellcheck disable=SC1091
+source ".drone/pipelines/default/runtime/binaries.sh"
+# shellcheck disable=SC1091
+source ".drone/pipelines/default/runtime/config.sh"
+
+# Trigger cf-acceptance-tests.
+bazel run //testing/acceptance_tests
+
+cf_acceptance_tests_pod_name() {
+  kubectl get pods --namespace "${KUBECF_NAMESPACE}" --output name 2> /dev/null | grep "acceptance-tests"
```

Comment:
Don't we have a selector we can use here?

Reply:
https://github.com/SUSE/kubecf/pull/243#discussion_r358526502